source/ocsp-support/ocsp-support.rst

```diff
@@ -100,9 +100,12 @@ invalid, the driver SHOULD end the connection.
     attempt to validate the status of the unvalidated certificates
     using the cached CRLs.
 
-7.  If the server’s certificate remains unvalidated and that certificate
-    has an OCSP endpoint, the driver SHOULD reach out to the OCSP
-    endpoint specified and attempt to validate that certificate.
+7.  If the server's certificate remains unvalidated and that certificate
+    has a list of OCSP responder endpoints, the driver SHOULD send HTTP requests
+    to the responders in parallel. The first valid response that concretely marks
+    the certificate status as good or revoked should be used. A five-second timeout
+    MUST be used for the requests. The status for a response should only be checked
```

Comment:
Perhaps "SHOULD" since we're using SHOULD like "MUST" in this section due to the inability to mandate MUST for drivers like C#/.NET (and possibly others)?

Reply:
Agreed. I had it as SHOULD to match the previous version, but I've changed it to MUST.

Reply:
Oops, sorry I was unclear! I was advocating for the use of SHOULD over MUST for this part of the spec since we are unable to use the word "MUST" due to drivers that don't have fine grained control over their OCSP behavior. 

In other words:
"...has a list of OCSP responder endpoints, the driver SHOULD send HTTP requests...A five-second timeout SHOULD be used for the requests"

Reply:
Ah, I misunderstood your previous comment. My reading was "if you're manually sending a request, it must use a 5-second timeout" but I can see why SHOULD is preferred here for drivers whose TLS libraries may not allow timeouts to be configured for OCSP requests. Changed to SHOULD in my latest commit.